src/common/signer.ts

```diff
@@ -98,7 +99,6 @@ export abstract class TurboDataItemAbstractSigner
         );
 
       case 'arweave':
-      default:
```

Comment:
removing the default in a typed switch makes the compiler force devs to update the switch as we add new tokens 👍